core/embed/rust/src/ui/layout/obj.rs

```diff
@@ -372,12 +384,12 @@ extern "C" fn ui_layout_touch_event(n_args: usize, args: *const Obj) -> Obj {
     unsafe { util::try_with_args_and_kwargs(n_args, args, &Map::EMPTY, block) }
 }
 
-#[cfg(any(feature = "model_t1", feature = "model_tr"))]
+#[cfg(all(any(feature = "model_t1", feature = "model_tr", not(feature = "model_tt"))))]
```

Comment:
If I'm reading this correctly then `all` only has one argument and therefore does nothing?

Reply:
Yeah, I had a mismatch in `()`.

Replaced with `#[cfg(not(feature = "model_tt"))]` and `#[cfg(feature = "model_tt")]` as suggested